aopy/utils/base.py

```diff
@@ -859,4 +859,52 @@ def get_consecutive_days(dates):
             temp_consec_dates = [dates[i]]
     if len(temp_consec_dates) > 1:
         consecutive_dates.append(temp_consec_dates)
-    return consecutive_dates
\ No newline at end of file
+    return consecutive_dates
+
+def scale_data_by_p_value(data, p, k=100, p0=0.08):
+    '''
+    Scale data by a sigmoid function of p-value. Useful for visualizing data maps
+    generated with p-values, to emphasize significant values.
+
```

Comment:
Adding a paper link may be helpful (https://www.science.org/doi/full/10.1126/scitranslmed.aay4682).